tools/run_command.py

```diff
@@ -47,3 +47,4 @@ def print_usage():
         commands[sys.argv[1]]['func'](sys.argv[2:])
     except Exception as e:
         print_usage()
+        exit(1)
```

Comment:
To provide more information and to classify the error situation, how about this?
```python
    except KeyError:
        print_usage()
    except Exception as e:
        print(e)
        exit(1)
```

Reply:
The tool prints misuses of commands as programmatic error, such as "list index out of range" for running `./tools/run_command.py`. I believe it can be solved by using `add_subparsers` in `argparse`, I'll update the PR after finishing #220.

Reply:
@bbrto21 
I've refactored the code to use subparsers, please review the changes when possible.